doc/source/theory/unres_cfd-dem/unres_cfd-dem.rst

```diff
@@ -0,0 +1,150 @@
+
+Models A and B differ from each other in the way the momentum equation is calculated. In Model A, we consider that pressure is in both phases, while for Model B the pressure is only in the fluid:
+
+Model A:
+
+.. math:: 
+    \rho_f \left ( \frac{\partial \left ( \varepsilon_f \mathbf{u} \right )}{\partial t} + \nabla \cdot \left ( \varepsilon_f \mathbf{u} \otimes \mathbf{u} \right ) \right ) = -\varepsilon \nabla p + \varepsilon \nabla \cdot \tau + \mathbf{F}_{pf}^A
+
+Model B:
+
+.. math:: 
+    \rho_f \left ( \frac{\partial \left ( \varepsilon_f \mathbf{u} \right )}{\partial t} + \nabla \cdot \left ( \varepsilon_f \mathbf{u} \otimes \mathbf{u} \right ) \right ) = -\nabla p + \nabla \cdot \tau + \mathbf{F}_{pf}^B
+
+where:
+
+* :math:`\rho_f` is the density of the fluid;
+* :math:`p` is the pressure;
+* :math:`\tau` is the shear stress;
+* :math:`\mathbf{F}_{pf}^A` and :math:`\mathbf{F}_{pf}^B` are the source terms representing the forces applied back in the fluid due to the interaction with particles for Models A and B, respectively.
+
+For Model A, since the pressure term corresponds to a 'fluid fraction of the pressure', we can write the interaction term as:
+
+.. math:: 
+    \mathbf{F}_{pf}^A = -\frac{1}{V_{\Omega}}\sum_{i}^{n_p}\left ( \mathbf{f}_{pf, i} - \mathbf{f}_{\nabla p, i} - \mathbf{f}_{\nabla \cdot \tau, i} \right )
+
+while for Model B, since the pressure is totally in the fluid, we write:
+
+.. math:: 
+    \mathbf{F}_{pf}^B = -\frac{1}{V_{\Omega}}\sum_{i}^{n_p}\left ( \mathbf{f}_{pf, i} \right )
+
+where :math:`n_p` is the number of particles inside the subdomain :math:`\Omega` with volume :math:`V_{\Omega}`.
+
+Lethe is capable of simulating unresolved CFD-DEM cases with both Models A and B (see the :doc:`../../parameters/unresolved_cfd-dem/cfd_dem` page of this guide).
+
+Void fraction
+--------------
+Determining the void fraction is an important step in unresolved CFD-DEM, as can be noted by the VANS equations and the drag models `[4] <http://dx.doi.org/10.1016/j.ces.2013.05.036>`_. In Lethe we apply the Particle Centroid Method (PCM) `[5] <https://doi.org/10.1002/aic.14421>`_, which is simple and the most popular method. It consists of tracking the position of the centroid of each particle and applying the total volume of the particle on the calculation of the void fraction of the cell. This means that in either of the following situations the void fraction of the colored cell is the same:
+
+.. image:: images/void_frac1.png
+.. image:: images/void_frac2.png
+
+PCM can be written as:
+
+.. math:: 
+    \varepsilon_f = 1 - \frac{\sum_{i}^{n_p} V_{p,i}}{\Omega}
+
+where :math:`n_p` is the number of particles with centroid inside the subdomain :math:`V_{\Omega}`.
+
+In finite Element, the void fraction must be projected to the mesh nodes so that one can assemble the system of equations. This is done by :math:`\mathcal{L}^2` projection `[6] <https://link.springer.com/book/10.1007/978-3-642-33287-6>`_:
```

Comment:
```suggestion
In the finite element method, the void fraction must be projected to the mesh nodes so that one can assemble the system of equations. This is done by :math:`\mathcal{L}^2` projection `[6] <https://link.springer.com/book/10.1007/978-3-642-33287-6>`_:
```